pkg/yurthub/cachemanager/cache_manager.go

```diff
@@ -600,3 +625,8 @@ func (cm *cacheManager) CanCacheFor(req *http.Request) bool {
 	}
 	return true
 }
+
+// DeleteInvalidKindFor is used to delete the invalid Kind(which is not registered in the cloud)
+func (cm *cacheManager) DeleteInvalidKindFor(gvr schema.GroupVersionResource) error {
```

Comment:
how about change the func name from `DeleteInvalidKindFor` to `DeleteKindFor`?

Reply:
The code has been updated, please review.